### ISSUE TITLE
Do not show we are unpacking the package

### ISSUE THREAD

**Issue by josevalim** [hexpm/hex#170]
In the snippet below:

```
* Getting tzdata (Hex package)
Checking package (https://s3-eu-west-1.amazonaws.com/s3-eu.hex.pm/tarballs/tzdata-0.5.5.tar)
Fetched package
Unpacked package tarball (/Users/jose/.hex/packages/tzdata-0.5.5.tar)
* Getting combine (Hex package)
Checking package (https://s3-eu-west-1.amazonaws.com/s3-eu.hex.pm/tarballs/combine-0.6.0.tar)
Fetched package
Unpacked package tarball (/Users/jose/.hex/packages/combine-0.6.0.tar)
* Getting ranch (Hex package)
Checking package (https://s3-eu-west-1.amazonaws.com/s3-eu.hex.pm/tarballs/ranch-1.2.0.tar)
Using locally cached package
Unpacked package tarball (/Users/jose/.hex/packages/ranch-1.2.0.tar)
* Getting cowlib (Hex package)
Checking package (https://s3-eu-west-1.amazonaws.com/s3-eu.hex.pm/tarballs/cowlib-1.0.2.tar)
Using locally cached package
Unpacked package tarball (/Users/jose/.hex/packages/cowlib-1.0.2.tar)
* Getting plug (Hex package)
Checking package (https://s3-eu-west-1.amazonaws.com/s3-eu.hex.pm/tarballs/plug-1.0.3.tar)
Using locally cached package
Unpacked package tarball (/Users/jose/.hex/packages/plug-1.0.3.tar)
```

I would say the "Unpacked package tarball (/Users/jose/.hex/packages/plug-1.0.3.tar)" is not relevant because it is the same for all packages. It will always be there and we can always inflect it will from the getting one. Is this true? If so, we should :scissors: it. :)

**Comment by lexmag** [comment]
I like :scissors:. Maybe we should have `trace` or `verbose` switch?

**Comment by josevalim** [comment]
For now I think we should just get rid of this line, the rest of the information is not that much specially it shows only when you are adding new packages.

**Comment by ericmj** [comment]
Agreed.

**Comment by lexmag** [comment]
Closed via e1411c11d5c80518aac671de1e042d275f879d5c.
